src/ccid_serial.c

```diff
@@ -833,6 +850,11 @@ status_t OpenSerialByName(unsigned int reader_index, char *dev_name)
 		unsigned char tx_buffer[] = { 0x02 };
 		unsigned char rx_buffer[50];
 		unsigned int rx_length = sizeof(rx_buffer);
+		
+		if (0 == strcasecmp(reader_name,"SEC1210"))
+		{
+			tx_buffer[0] = 0x06;
+		}
```

Comment:
when we sent 0x02 to get firmware version, 
"03 06 6B 01 00 00 00 00 00 00 00 00 02 6D"
we received
frame: "83 00 00 00 00 00 00 41 00 00" - "Command not supported."

when we sent 0x06 to get firmware version, 
"03 06 6B 01 00 00 00 00 00 00 00 00 06 69"
we received
frame: "83 01 00 00 00 00 00 01 00 00 22", "Everything OK."